chainerx_cc/chainerx/routines/manipulation.h

```diff
@@ -11,6 +12,11 @@
 
 namespace chainerx {
 
+enum class PadMode {
+    constant = 1,
+    edge,
```

Comment:
`edge` and `reflection` are not implemented.
How about removing them so that no one would accidently use them?
(Perhaps we can write TODO here to implement them)

Reply:
Sure, I have done that now.